cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -1565,6 +1561,110 @@ data MultiSigScript = RequireSignature (Hash PaymentKey)
                     | RequireMOf Int [MultiSigScript]
   deriving (Eq, Show)
 
+instance ToJSON MultiSigScript where
```

Comment:
Just a note that maybe we should move this `MultiSigScript` stuff into its own module at some point.

Reply:
I agree, but I'm waiting on @dcoutts to decide on the best way to break up the whole `Typed.hs` module